dashboard/src/actions/keyManagementActions.js

```diff
@@ -0,0 +1,98 @@
+import * as TYPES from "actions/types";
+
+import { DANGER, ERROR_MSG, SUCCESS } from "assets/constants/toastConstants";
+
+import API from "../utils/axiosInstance";
+import { showToast } from "./toastActions";
+import { uriTemplate } from "utils/helper";
+
+export const getAPIkeysList = () => async (dispatch, getState) => {
```

Comment:
Yeah, I am interested, what is the difference between the 2 approaches and when would you use onv over the other? Is an extra function call only useful if there is a parameter to pass?

Reply:
> Is an extra function call only useful if there is a parameter to pass?

Just so.  The function call causes the value to be generated at runtime instead of at...whatever Javascript's equivalent of "compile-time" is.  This is useful if the value needs to have parameters inserted into it; otherwise, it's just wasted overhead.